components/events/summer-school/FaqSection.vue

```diff
@@ -0,0 +1,323 @@
+    </cv-accordion>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import { SegmentData } from '~/constants/appLinks'
+
+// GeneralLink does not fit. 'label' is not necessary
+interface TrackedLink {
+  url: string,
+  segment: SegmentData
+}
+
+@Component
+export default class FaqSection extends Vue {
+  khanacademy: TrackedLink = {
+    url: 'https://www.khanacademy.org/math/precalculus/x9e81a4f98389efdf:matrices/x9e81a4f98389efdf:multiplying-matrices-by-matrices/v/matrix-multiplication-intro',
+    segment: { action: 'events-summer-school > faq > khanacademy-matrix-multiplication-intro' }
+  }
+
+  pythonAndJupyter: TrackedLink = {
+    url: 'https://qiskit.org/textbook/ch-prerequisites/python-and-jupyter-notebooks.html',
+    segment: { action: 'events-summer-school > faq > textbook-python-and-jupyter-notebooks' }
+  }
+
+  linearAlgebra: TrackedLink = {
+    url: 'https://qiskit.org/textbook/ch-prerequisites/linear_algebra.html',
+    segment: { action: 'events-summer-school > faq > qiskit-linear-algebra' }
+  }
+
+  qCEducators: TrackedLink = {
+    url: 'https://quantum-computing.ibm.com/programs/educators',
+    segment: { action: 'events-summer-school > faq > ibm-quantum-educators-program' }
+  }
+
+  mailtoQCEducators: TrackedLink = {
+    url: 'mailto:quantum.education@us.ibm.com',
+    segment: { action: 'events-summer-school > faq > mailto-quantum-educators-program' }
+  }
+
+  twitterQiskit: TrackedLink = {
+    url: 'https://twitter.com/qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-twitter' }
+  }
+
+  twitterIBMResearch: TrackedLink = {
+    url: 'https://twitter.com/IBMResearch',
+    segment: { action: 'events-summer-school > faq > ibmresearch-twitter' }
+  }
+
+  githubQiskit: TrackedLink = {
+    url: 'https://github.com/Qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-github' }
+  }
+
+  mediumQiskit: TrackedLink = {
+    url: 'https://medium.com/qiskit',
+    segment: { action: 'events-summer-school > faq > qiskit-medium' }
+  }
+
+  qiskitSubscribe: TrackedLink = {
+    url: 'https://qisk.it/sub',
+    segment: { action: 'events-summer-school > faq > qiskit-youtube-subscribe' }
+  }
+
+  joinqiskitslack: TrackedLink = {
+    url: 'https://ibm.co/joinqiskitslack',
+    segment: { action: 'events-summer-school > faq > qiskit-slack-join' }
+  }
+
+  qiskitEvents: TrackedLink = {
+    url: 'https://qiskit.org/events',
+    segment: { action: 'events-summer-school > faq > qiskit-org-events' }
+  }
+
+  mailtoQiskit: TrackedLink = {
+    url: 'mailto:qiskit.events@us.ibm.com',
+    segment: { action: 'events-summer-school > faq > mailto-qiskit-events' }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "~/assets/scss/layout/_copy.scss";
+
+.faq-section {
+  // stylelint-disable selector-pseudo-element-no-unknown
+  ::v-deep {
+  // stylelint-enable selector-pseudo-element-no-unknown
```

Comment:
We're already disallowing this rule globally in the Stylelint config in this PR.

```suggestion
  ::v-deep {
```